src/apm.coffee

```diff
@@ -61,7 +61,7 @@ module.exports =
     process.env.ATOM_API_URL ? 'https://atom.io/api'
 
   getNodeVersion: ->
-    process.env.ATOM_NODE_VERSION ? '0.22.0'
+    process.env.ATOM_NODE_VERSION ? require('package.json').electronVersion
```

Comment:
Should this default to the `electronVersion` in Atom's `package.json` instead? So we don't have to track it in multiple places.